Core/bin/getenv.sh

```diff
@@ -155,6 +168,22 @@ mango_stop() {
   rm -f "$mango_paths_pid_file"
 }
 
+print_value_source_message() {
+  if [ "$1" = "$2" ]; then
+    echo "++ $3 mango.properties file property $4"
+  else
+    echo "++++++ $3 default value $2"
+  fi
+}
+
+print_ma_keystore_value_source_message() {
+  if [ "$1" = "$2" ]; then
+    echo "++ MA_KEYSTORE used $1"
+  else
+    echo "++++++ $3 default value $2 as file $1 was not found."
```

Comment:
This log entry can be misleading. If the default path is valid, we shouldn't tell the user the file was not found.
```suggestion
    if [ -e $1 ]
    then
        echo "++++++ $3 default value $2"
    else
        echo "++++++ $3 default value $2 as file $1 was not found."
    fi
```